### PR TITLE
Fix: prevent queryDisplyText in QueryRowHeader from overflowing

### PR DESCRIPTION
**Pull request by Elfo404**
**What this PR does / why we need it**:

prevents `collapsedText` from overflowing horizontally. This is an alternative approach to https://github.com/grafana/grafana/pull/39419 (reverted in this PR) to avoid an issue when, in alerting UI, the timepicker was not being shown anymore.

### Alerting UI
Screenshot 2021-10-06 at 16 38 00

### Dashboard edit
Screenshot 2021-10-06 at 16 37 16

### Explore
Screenshot 2021-10-06 at 16 36 42

### CODE REVIEW

File: public/app/core/components/QueryOperationRow/QueryOperationRow.tsx [hunk 1]
```diff
@@ -164,6 +176,7 @@ const getQueryOperationRowStyles = stylesFactory((theme: GrafanaTheme) => {
       color: ${theme.colors.textBlue};
       margin-left: ${theme.spacing.sm};
       overflow: hidden;
+      text-overflow: ellipsis;
```
Comment (Elfo404) [inline public/app/core/components/QueryOperationRow/QueryOperationRow.tsx:179]: This is unrelated, just a small fix for `title` used in transformations:

Before:
Screenshot 2021-10-06 at 16 46 40

After:
Screenshot 2021-10-06 at 16 46 56

**Review by ifrost — APPROVED** [review]
Sweet 🚀

**Review by peterholmberg — APPROVED** [review]
Awesome Gio 🥳
